src/Twig/Node/LayoutNode.php

```diff
@@ -0,0 +1,57 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * CakePHP(tm) : Rapid Development Framework (https://cakephp.org)
+ * Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
+ * Copyright (c) 2014 Cees-Jan Kiewiet
```

Comment:
I don't think copyright of previous author is required in new files.

Reply:
removed